Resource/randomStringFactory.js

```diff
@@ -47,15 +47,15 @@ function randomStringGeneratorFactory() {
 
   return {
     randomString: function( {
-        length = 12,

```

Comment:
using `[instance]("somestring")` takes the length property of the first parameter (a string), so rename